tree-view-sample/src/testView.ts

```diff
@@ -1,9 +1,31 @@
 import * as vscode from 'vscode';
 
-export class TestView {
+export class TestView implements vscode.TreeDataProvider<Node>, vscode.DragAndDropController<Node> {
+	private _onDidChangeTreeData: vscode.EventEmitter<Node[] | undefined> = new vscode.EventEmitter<Node[] | undefined>();
+	public onDidChangeTreeData: vscode.Event<Node[] | undefined> = this._onDidChangeTreeData.event;
```

Comment:
The vscode API doesn't yet support having an array in `onDidChangeTreeData`. Is a PR for that still comming?